pmemcheck/pmc_main.c

```diff
@@ -842,6 +846,30 @@ read_cache_line_size(void)
     return ret_val;
 }
 
+/**
+* \brief Tries to register a file mapping.
+* \return Returns -1 on errors, 0  otherwise.
+*/
```

Comment:
1) Please, add the description of arguments.
2) ... 0 on success, -1 otherwise.